### ISSUE TITLE
Implement queue system for ticket requests

### ISSUE THREAD

**Issue by Jonahbui** [Jonahbui/Fender#17]
The queue system should process each ticket (a delivery request) one at a time as long as there are some present. Another ticket should be executed only after the previous ticket has been completely finished.

Outline of code
```
WHILE there are tickets in the queue:
  WHILE there is not a ticket currently being processed:
    GET the next ticket to process
    SET that ticket as the current ticket
    PROCESS the current ticket
    AWAIT user confirmation that request has been complete
    IF the user fails to confirm after X amount of time, PROCESS the next request automatically
```
